code/modules/surgery/tools.dm

```diff
@@ -292,7 +293,8 @@
 		icon_state = "retractor_a"
 
 /obj/item/retractor/advanced/examine()
-	. += " It resembles a retractor[tool_behaviour == TOOL_RETRACTOR ? "retractor" : "hemostat"]."
+	. = ..()
+	. += " It resembles a [tool_behaviour == TOOL_RETRACTOR ? "retractor" : "hemostat"]."

```

Comment:
considering these defines are already strings, this doesn't really need a compare.

Reply:
:monkey: :speech_balloon: " :gorilla: "